typescript-practice/src/ts/helpers/interface.ts

```diff
@@ -0,0 +1,4 @@
+export interface dataObj {
```

Comment:
First, model should use CamelCase naming
Second, interface should not in helper

Reply:
Fixed.